cmp/grid/impl/ColumnHeader.js

```diff
@@ -77,7 +77,8 @@ export const columnHeader = hoistCmp.factory({
         if (isDesktop && isUndefined(xhColumn?.headerTooltip)) {
             onMouseEnter = ({target: el}) => {
                 if (el.offsetWidth < el.scrollWidth) {
-                    el.setAttribute('title', headerName);
+                    const title = isObject(headerName) ? xhColumn?.chooserName : headerName;
```

Comment:
Seems like this should also be isString(headerName)?  headerName : props.displayName

A lot of this code would be clearer if we referred to the local variable as `headerElem` as that is the most general description of iut.